src/test/rpc/Transaction_test.cpp

```diff
@@ -791,7 +861,8 @@ class Transaction_test : public beast::unit_test::suite
         testCTIDValidation(features);
         testCTIDRPC(features);
         test::jtx::forAllApiVersions(
-            std::bind_front(&Transaction_test::testRequest, this, features));
+            std::bind_front(&Transaction_test::testRequest, this, features),
+            std::bind_front(&Transaction_test::testBinaryRequest, this));
```

Comment:
Since `testBinaryRequest` doesn't take a `features` parameter, it probably makes sense to call it from `run()`.

Reply:
Thanks, done